src/SubscribeMe/Interface/SubscriberInterface.php

```diff
@@ -21,7 +22,15 @@ public function setContactListId(?string $contactListId): SubscriberInterface;
      * @param array       $options
      * @param UserConsent[] $userConsents
      *
-     * @return mixed Contact ID if succeeded or false
+     * @return bool|int Contact ID if succeeded or false
      */
-    public function subscribe(string $email, array $options, array $userConsents = []);
+    public function subscribe(string $email, array $options, array $userConsents = []): bool|int;
+
+    /**
+     * @param array<EmailAddress> $emails
+     * @param array $variables
+     * @param string $templateEmail
+     * @return string
+     */
+    public function sendTransactionalEmail(array $emails, array $variables, string $templateEmail): string;
```

Comment:
```suggestion
    public function sendTransactionalEmail(array $emails, array $variables, string|int $emailTemplateId): string;
```

Template ID may be a string (UUID) or int